tests/bgp/test_bgp_4-byte_as_trans.py

```diff
@@ -0,0 +1,290 @@
+    -c "no bgp default ipv4-unicast" \
+    -c "bgp bestpath as-path multipath-relax" \
+    -c "neighbor {} peer-group" \
+    -c "neighbor {} peer-group" \
+    -c "neighbor {} remote-as {}" \
+    -c "neighbor {} peer-group {}" \
+    -c "neighbor {} description {}" \
+    -c "neighbor {} timers 3 10" \
+    -c "neighbor {} timers connect 10" \
+    -c "neighbor {} remote-as {}" \
+    -c "neighbor {} peer-group {}" \
+    -c "neighbor {} description {}" \
+    -c "neighbor {} timers 3 10" \
+    -c "neighbor {} timers connect 10" \
+    -c "neighbor {} remote-as {}" \
+    -c "neighbor {} peer-group {}" \
+    -c "neighbor {} description {}" \
+    -c "neighbor {} timers 3 10" \
+    -c "neighbor {} timers connect 10" \
+    -c "neighbor {} remote-as {}" \
+    -c "neighbor {} peer-group {}" \
+    -c "neighbor {} description {}" \
+    -c "neighbor {} timers 3 10" \
+    -c "neighbor {} timers connect 10" \
+    -c "address-family ipv4 unicast" \
+    -c "network {}" \
+    -c "neighbor {} soft-reconfiguration inbound" \
+    -c "neighbor {} route-map FROM_BGP_PEER_V4 in" \
+    -c "neighbor {} route-map TO_BGP_PEER_V4 out" \
+    -c "neighbor {} activate" \
+    -c "neighbor {} activate" \
+    -c "maximum-paths 64" \
+    -c "exit-address-family" \
+    -c "address-family ipv6 unicast" \
+    -c "network {}" \
+    -c "neighbor {} soft-reconfiguration inbound" \
+    -c "neighbor {} route-map FROM_BGP_PEER_V6 in" \
+    -c "neighbor {} route-map TO_BGP_PEER_V6 out" \
+    -c "neighbor {} activate" \
+    -c "neighbor {} activate" \
+    -c "maximum-paths 64" \
+    -c "exit-address-family" \
+    '.format(setup['cli_options'], setup['dut_asn'], dut_4byte_asn, setup['dut_bgp_id'],
+             setup['peer_group_v4'], setup['peer_group_v6'], setup['neigh_ip_v4'], neighbor_4byte_asn,
+             setup['neigh_ip_v4'], setup['peer_group_v4'], setup['neigh_ip_v4'], setup['neigh1'], setup['neigh_ip_v4'],
+             setup['neigh_ip_v4'], setup['neigh_ip_v6'], neighbor_4byte_asn, setup['neigh_ip_v6'],
+             setup['peer_group_v6'], setup['neigh_ip_v6'], setup['neigh1'], setup['neigh_ip_v6'], setup['neigh_ip_v6'],
+             setup['neigh2_ip_v4'], setup['neigh2_asn'], setup['neigh2_ip_v4'], setup['peer_group_v4'],
+             setup['neigh2_ip_v4'], setup['neigh2'], setup['neigh2_ip_v4'], setup['neigh2_ip_v4'],
+             setup['neigh2_ip_v6'], setup['neigh2_asn'], setup['neigh2_ip_v6'], setup['peer_group_v6'],
+             setup['neigh2_ip_v6'], setup['neigh2'], setup['neigh2_ip_v6'], setup['neigh2_ip_v6'],
+             setup['dut_ipv4_network'], setup['peer_group_v4'], setup['peer_group_v4'], setup['peer_group_v4'],
+             setup['neigh_ip_v4'], setup['neigh2_ip_v4'], setup['dut_ipv6_network'], setup['peer_group_v6'],
+             setup['peer_group_v6'], setup['peer_group_v6'], setup['neigh_ip_v6'], setup['neigh2_ip_v6'])
+    logger.debug(setup['duthost'].shell(cmd, module_ignore_errors=True))
+
+    cmd = 'vtysh \
```

Comment:
can you add the `cli_options = "-n " + duthost.get_namespace_from_asic_id(asic_index)` here also? T2 neighbor could be multi-asic